test/functional/Jsr292/playlist.xml

```diff
@@ -102,6 +102,32 @@
 		</subsets>
 	</test>
 
+	<test>
+		<testCaseName>jsr292Test_jdk12</testCaseName>
+		<variations>
+				<variation>NoOptions</variation>
+				<variation>Mode195</variation>
```

Comment:
is Mode195 a JIT count=0 mode?  These tests will need to be run both normally and ensuring they've been jitted.

Reply:
`-Xjit` is set for this mode